src/code/engine_library/calculateBodySurfaceArea.m

```diff
@@ -13,12 +13,12 @@
 
 
 switch WSettings.BSA_calculationmethode
-
+    
     case 'DuBois'        
-        BSA = (0.007184.*(height.*10).^0.725 .*weight.^0.425).*100;
+        BSA = (0.007184.*(height.*10).^0.725 .*weight.^0.425);
         
```

Comment:
here BSA is calculated in m^2

Reply:
Yes that's what is should be.
For added Parameter no unit conversion is possible, that means they have to be added in the final unit shown in the plots. That would be for BSA m^2.